ChaosMod/Effects/db/Peds/PedsSpawnAngrySimeon.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+	Effect by Reguas
+*/
+
+#include <stdafx.h>
+
+static void OnStart()
+{
+	static constexpr Hash modelHashSimeon = 0xC0937202;
+	static constexpr Hash modelHashLamar = 0x65B93076;
+	static constexpr Hash modelHashFranklin = 0x9B22DBAF;
+
+	Ped playerPed = PLAYER_PED_ID();
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, false);
+
+	static const Hash playerGroup = GET_HASH_KEY("PLAYER");
+
+	Hash relationshipGroup;
```

Comment:
Could this be replaced with „ CreateHostilePed“?

Reply:
It possibly could, but in that case Lamar/Franklin refuses to get into passenger seat and drive player's vehicle.